python_modules/dagster/dagster_tests/core_tests/blueprint_tests/yaml_files/definition_set_nested.yaml

```diff
@@ -0,0 +1,13 @@
+contents:
+  - key: asset1
+  - contents:
+      - key: asset2
+    metadata:
+      foo: bar
+    tags:
+      inner: tag
+
+metadata:
```

Comment:
Having these properties at the top level feels weird because they only apply to assets but it seems like they would apply  to everything. Thoughts on nesting them under something like a `default_asset_attributes` property?